README.md

```diff
@@ -1 +1,33 @@
-# gci-cn
\ No newline at end of file
+<p align="center">
+  <img width="120" src="https://codein.withgoogle.com/static/favicon/apple-icon-120x120.png" alt="logo" />
+</p>
+
+[![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/Gsoc-cn/Lobby)
+
+Welcome to **Open Source** world! Google Code In (GCI) is a program which provides secondary school students with great opportunities to contribute to open source projects. You can go to the [official website](https://codein.withgoogle.com/) for detailed information (you may need to use VPN if you are in mainland China).
+
+欢迎来到开源世界！Google Code In (简称 GCI，暂无中文翻译) 是中学生参加开源项目的一个非常好的机会。更多资料请移步[官网](https://codein.withgoogle.com/)（中国大陆地区可能需要科学上网）。
+
+You can participate in this activity as long as you are a pre-university student ages 13 to 17. Participants get to work on real software and win prizes from t-shirts to a trip to Google Headquarter!
+
+13至17岁的中学生都可以参加这个活动。你将为开源组织做贡献，并有机会赢得丰厚的奖品，奖品小到纪念衫，大到一次参观google总部的机会（含双人往返美国加州山景城机票、食宿）。
+
+You do not need to be a programmer -- you don’t have to know how to code at all! All contestants are welcome to choose non-coding tasks that may better fit their interests/skills. Non-coding tasks include documentation, training, outreach, research, design, and quality assurance (e.g., find bugs and report them, etc.) tasks.
```

Comment:
This paragraph is copied from GCI official website. I hope there is no copyright issue ^^